### PR TITLE
turn repository into a mkdocs website to host training materials

### PR DESCRIPTION
**Pull request by taylorreiter**
This pull request creates a webpage https://arcadia-science.github.io/arcadia-computational-training that will be a user-friendly site to host training materials. The site is built with mkdocs. I've included local rendering instructions in the README. There's still more content to add, but I think it's ready to go live.

One things I wanted to check about is whether we should have https://arcadia-science.github.io resolve to anything other than a 404? This repo and Elizabeth's https://arcadia-science.github.io/arcadia-metagenomics-workflows/ both are hosted at the domain `arcadia-science.github.io`...just thought we maybe should brainstorm around the right approach here.

### CODE REVIEW

File: docs/workshops/overview.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Arcadia Workshops
```
Comment (mertcelebi) [inline docs/workshops/overview.md:1]: Are these different from AUG sessions? Maybe like the Github workshop?

File: mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+site_name: Arcadia Science Computational Training
+site_url: http://arcadia-science.github.io/arcadia-computational-training
+repo_name: arcadia-computational-training
+repo_url: https://github.com/arcadia-science/arcadia-computational-training
+edit_uri: ""
+
+copyright: 'Copyright &copy; 2022 <a href="https://www.arcadia.science/">Arcadia Science</a>'
```
Comment (mertcelebi) [inline mkdocs.yml:7]:
```suggestion
copyright: 'Copyright &copy; 2022 <a href="https://www.arcadiascience.com">Arcadia Science</a>'
```

File: docs/arcadia-users-group/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Arcadia Users' Group training materials
+
+This is the location for training materials associated with tutorials given during Arcadia Users' Group. 
+
+## Contributing
```
Comment (mertcelebi) [inline docs/arcadia-users-group/README.md:5]: Maybe some of this should live in the other contributing doc?

**Comment by taylorreiter** [comment]
> Approving with a couple comments:
> 
>     * Note for future: it'd be great if you could separate the mkdocs work from the lecture changes.
> 
ok! Will do
>     * Could you fix the alignment in the logo? Happy to quickly design a logo as well if that'd be easier for you. I know it's for an internal site, but details matter.
> 
I'm happy to update it! What would you like fixed with the alignment though? I was trying to be consistent with here: https://arcadia-science.github.io/arcadia-metagenomics-workflows/

>     * For the landing page for the root domain question you asked in the PR description: perhaps it can just be a simple landing page that lists sub-sections? 404 doesn't seem ideal. Or it could redirect to one of the sub-sections? Also let me know if need a shorter URL like: `training.arcadiascience.com`. I can help with that if needed.

 I think a landing page with sub-sections would be great for now. I don't think we need a shorter url right now but i'll let you know if that changes, thank you!

**Comment by taylorreiter** [comment]
I've updated the image, let me know if you had something else in mind!

**Pull request merged**